modules/http/src/main/java/org/glassfish/grizzly/http/util/CookieHeaderParser.java

```diff
@@ -248,6 +248,15 @@ public static boolean isToken(int c) {
         }
     }
 
+    public static boolean isText(int c) {
+        // Fast for correct values, slower for incorrect ones
+        try {
+            return isText[c];
+        } catch (ArrayIndexOutOfBoundsException ex) {
```

Comment:
Isn't it better to check if you are in the 0 - isText.length range than "exception driven code"?

Reply:
Yes. I agree that checking the range is better than throwing an exception. The reason for doing this is to maintain similarity with other existing code.

Reply:
Great opportunity to stop this madness.

Reply:
But for the comment:
https://github.com/eclipse-ee4j/grizzly/blob/4316ca1adbaa00341d142d8ae571932294511a96/modules/http/src/main/java/org/glassfish/grizzly/http/util/CookieHeaderParser.java#L33-L34
need to dig at origin
https://github.com/eclipse-ee4j/grizzly/blob/4316ca1adbaa00341d142d8ae571932294511a96/modules/http/src/main/java/org/glassfish/grizzly/http/util/CookieHeaderParser.java#L36

Reply:
> The reason for doing this is to maintain similarity with other existing code.

:+1:
\+ to investigate the change in both places later (perhaps never)